go-controller/pkg/ovn/master.go

```diff
@@ -416,6 +418,51 @@ func (oc *DefaultNetworkController) addNode(node *kapi.Node) ([]*net.IPNet, erro
 		return nil, err
 	}
 
+	// For live migrated pods the switch manager has to be refill in
+	// case the node has take over the vm subnet
+	liveMigratablePods, err := kubevirt.FindLiveMigratablePods(oc.watchFactory)
+	if err != nil {
+		return nil, err
+	}
+	for _, liveMigratablePod := range liveMigratablePods {
+		isStale, err := kubevirt.IsMigratedSourcePodStale(oc.watchFactory, liveMigratablePod)
+		if err != nil {
+			return nil, err
+		}
+
+		// We care only for Running virt-launcher pods
+		if isStale {
+			continue
+		}
+
+		annotation, err := util.UnmarshalPodAnnotation(liveMigratablePod.Annotations, ovntypes.DefaultNetworkName)
+		if err != nil {
+			continue
+		}
+		ok := false
+		switchName, ok := oc.lsManager.FindSwitchBySubnets(annotation.IPs)
+		// If switch manager contains the pod subnet we have to
+		// do allocatePodIP
+		if !ok {
+			continue
+		}
+		if _, err := oc.allocatePodIPsForSwitch(liveMigratablePod, annotation, ovntypes.DefaultNetworkName, switchName); err != nil {
+			return nil, err
+		}
+
+		// If vm is at a remote zone the remote zone routes need to be
+		// configured
+		liveMigratablePodNode, err := oc.watchFactory.GetNode(liveMigratablePod.Spec.NodeName)
+		if err != nil {
+			return nil, err
+		}
+		if !oc.isLocalZoneNode(liveMigratablePodNode) {
+			if err := kubevirt.EnsureRemoteZonePodAddressesToNodeRoute(oc.controllerName, oc.watchFactory, oc.nbClient, liveMigratablePod, ovntypes.DefaultNetworkName); err != nil {
+				return nil, err
+			}
+		}
```

Comment:
I don't think we need this. This has nothing to do with the fact that a node is added. It could have something to do with the fact that a zone is added, but if that is the case, there will be a new controller which will process pod events and this will be handled there.

Reply:
Looks like ensurePod is going to be call at the new node, so we don't need this. I will remove it.